core/include/detray/geometry/shapes/annulus2D.hpp

```diff
@@ -124,6 +124,32 @@ class annulus2D {
                 (r_mod2 <= maxR_tol * maxR_tol));
     }
 
+    /// @brief Measure of the shape: Area (not implemented!)
+    ///
+    /// @param bounds the boundary values for this shape
+    ///
+    /// @returns the stereo annulus area on the plane.
+    template <template <typename, std::size_t> class bounds_t,
+              typename scalar_t, std::size_t kDIM,
+              typename std::enable_if_t<kDIM == e_size, bool> = true>
+    DETRAY_HOST_DEVICE constexpr scalar_t measure(
+        const bounds_t<scalar_t, kDIM> &bounds) const {
+        return area(bounds);
+    }
+
+    /// @brief The area of a the shape
+    ///
+    /// @param bounds the boundary values for this shape
+    ///
+    /// @returns the stereo annulus area.
+    template <template <typename, std::size_t> class bounds_t,
+              typename scalar_t, std::size_t kDIM,
+              typename std::enable_if_t<kDIM == e_size, bool> = true>
+    DETRAY_HOST_DEVICE constexpr scalar_t area(
+        const bounds_t<scalar_t, kDIM> &) const {
+        return detail::invalid_value<scalar_t>();
```

Comment:
I don't think this is invalid

Reply:
It is not implemented at the moment, hence invalid

Reply:
Then adding a comment would be appreciated